<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Mechanistic AI in Genomics Lab </title> <meta name="author" content=" "> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://maigelab.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Mechanistic AI in Genomics Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/books/">bookshelf</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/blog/">blog</a> </div> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography"></h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_isometries_2011" class="col-sm-8"> <div class="title">Isometries of optimal pseudo-Riemannian metrics</div> <div class="author"> Brian Clarke </div> <div class="periodical"> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We give a concise proof that large classes of optimal (constant curvature or Einstein) pseudo-Riemannian metrics are maximally symmetric within their conformal class.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_decompositions_2008" class="col-sm-8"> <div class="title">Decompositions of the tensor products of irreducible sl(2)-modules in characteristic 3</div> <div class="author"> Brian Clarke </div> <div class="periodical"> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>We completely describe the decompositions (into indecomposable submodules) of the tensor products of irreducible sl(2)-modules in characteristic 3. The answer resembles analogous decompositions for the Lie superalgebra sl(1\textbar1).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_geodesics_2013" class="col-sm-8"> <div class="title">Geodesics, distance, and the CAT(0) property for the manifold of Riemannian metrics</div> <div class="author"> Brian Clarke </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/s00209-012-0996-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Given a fixed closed manifold M, we exhibit an explicit formula for the distance function of the canonical L2 Riemannian metric on the manifold of all smooth Riemannian metrics on M. Additionally, we examine the (metric) completion of the manifold of metrics with respect to the L2 metric and show that there exists a unique minimal path between any two points. This path is also given explicitly. As an application of these formulas, we show that the metric completion of the manifold of metrics is a CAT(0) space.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_riemannian_2011" class="col-sm-8"> <div class="title">The Riemannian L2 topology on the manifold of Riemannian metrics</div> <div class="author"> Brian Clarke </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/s10455-010-9227-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We study the manifold of all Riemannian metrics over a closed, finite-dimensional manifold. In particular, we investigate the topology on the manifold of metrics induced by the distance function of the L2 Riemannian metric‚Äîso-called because it induces an L2 topology on each tangent space. It turns out that this topology on the tangent spaces gives rise to an L1-type topology on the manifold of metrics itself. We study this new topology and its completion, which agrees homeomorphically with the completion of the L2 metric. We also give a user-friendly criterion for convergence (with respect to the L2 metric) in the manifold of metrics.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_conformal_2013" class="col-sm-8"> <div class="title">Conformal deformations of the Ebin metric and a generalized Calabi metric on the space of Riemannian metrics</div> <div class="author"> Brian Clarke and Yanir A. Rubinstein </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.anihpc.2012.07.003" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We consider geometries on the space of Riemannian metrics conformally equivalent to the widely studied Ebin L2 metric. Among these we characterize a distinguished metric that can be regarded as a generalization of Calabi ºs metric on the space of K√§hler metrics to the space of Riemannian metrics, and we study its geometry in detail. Unlike the Ebin metric, its geodesic equation involves non-local terms, and we solve it explicitly by using a constant of the motion. We then determine its completion, which gives the first example of a metric on the space of Riemannian metrics whose completion is strictly smaller than that of the Ebin metric.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_completion_2013" class="col-sm-8"> <div class="title">The completion of the manifold of Riemannian metrics</div> <div class="author"> Brian Clarke </div> <div class="periodical"> <em></em> </div> <div class="periodical"> Publisher: Lehigh University </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.4310/jdg/1361800866" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We give a description of the completion of the manifold of all smooth Riemannian metrics on a fixed smooth, closed, finitedimensional, orientable manifold with respect to a natural metric called the L2L2L^2 metric. The primary motivation for studying this problem comes from Teichm√ºller theory, where similar considerations lead to a completion of the well-known Weil-Petersson metric. We give an application of the main theorem to the completions of Teichm√ºller space with respect to a class of metrics that generalize the Weil-Petersson metric.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_ricci_2013" class="col-sm-8"> <div class="title">Ricci flow and the metric completion of the space of K√§hler metrics</div> <div class="author"> Brian Clarke and Yanir A. Rubinstein </div> <div class="periodical"> <em></em> </div> <div class="periodical"> Publisher: Johns Hopkins University Press </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1353/ajm.2013.0051" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>, We consider the space of K√§hler metrics as a Riemannian submanifold of the space of Riemannian metrics, and study the associated submanifold geometry. In particular, we show that the intrinsic and extrinsic distance functions are equivalent. We also determine the metric completion of the space of K√§hler metrics, making contact with recent generalizations of the Calabi-Yau Theorem due to Dinew and Guedj-Zeriahi. As an application, we obtain a new analytic stability criterion for the existence of a K√§hler-Einstein metric on a Fano manifold in terms of the Ricci flow and the distance function. We also prove that the K√§hler-Ricci flow converges as soon as it converges in the metric sense.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_gaussian_2015" class="col-sm-8"> <div class="title">Gaussian measures on the of space of Riemannian metrics</div> <div class="author"> Brian Clarke, Dmitry Jakobson, Niky Kamran, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Lior Silberman, Jonathan Taylor' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/s40316-015-0037-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We introduce Gaussian-type measures on the manifold of all metrics with a fixed volume form on a compact Riemannian manifold of dimension \{}ge \\‚â•3. For this random model we compute the characteristic function for the \≈Å^2\\L2(Ebin) distance to the reference metric. In the Appendix, we study Lipschitz-type distance between Riemannian metrics, and give applications to the diameter, eigenvalue and volume entropy functionals.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_metric_2010" class="col-sm-8"> <div class="title">The metric geometry of the manifold of Riemannian metrics over a closed manifold</div> <div class="author"> Brian Clarke </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/s00526-010-0323-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We prove that the L2 Riemannian metric on the manifold of all smooth Riemannian metrics on a fixed closed, finite-dimensional manifold induces a metric space structure. As the L2 metric is a weak Riemannian metric, this fact does not follow from general results. In addition, we prove several results on the exponential mapping and distance function of a weak Riemannian metric on a Hilbert/Fr√©chet manifold. The statements are analogous to, but weaker than, what is known in the case of a Riemannian metric on a finite-dimensional manifold or a strong Riemannian metric on a Hilbert manifold.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="rohbeck_bicycle_2024" class="col-sm-8"> <div class="title">Bicycle: Intervention-Based Causal Discovery with Cycles</div> <div class="author"> Martin Rohbeck, Brian Clarke, Katharina Mikulik, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Alexandra Pettet, Oliver Stegle, Kai Ueltzh√∂ffer' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>In Proceedings of the Third Conference on Causal Learning and Reasoning</em>, </div> <div class="periodical"> ISSN: 2640-3498 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>While a growing number of algorithms for causal discovery of directed acyclic graphs from observational and interventional data have been proposed, the robust identification of cyclic causal graphs in particular remains an open problem. Solutions to this challenge would have a considerable impact in various application domains, including single-cell genomics, where gene regulatory networks are known to contain feedback loops. Recent work has shown promise to address this challenge by describing the expression states in a population of cells as the steady-state solution of a stochastic dynamical system. However, this formulation cannot account for information on interventions in the population, and consequently, it ignores the associated causal inductive biases, which are key assets to obtain meaningful results and improve identifiability. In this work, we propose a novel method, Bicycle, which (i) infers cyclic causal relationships from i.i.d. data, (ii) explicitly accounts for information on the perturbation state of cells by a realization of the independent causal mechanism principle and (iii) models causal effects in a latent space rather than on observed data. We benchmark Bicycle in the context of existing approaches, demonstrating improved recovery of simulated causal graphs and improved out-of-distribution prediction performance on unseen perturbations in real single-cell datasets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="clarke_integration_2024" class="col-sm-8"> <div class="title">Integration of variant annotations using deep set networks boosts rare variant association testing</div> <div class="author"> Brian Clarke, Eva Holtkamp, Hakime √ñzt√ºrk, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Marcel M√ºck, Magnus Wahlberg, Kayla Meyer, Felix Munzlinger, Felix Brechtmann, Florian R. H√∂lzlwimmer, Jonas Lindner, Zhifen Chen, Julien Gagneur, Oliver Stegle' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41588-024-01919-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract Rare genetic variants can have strong effects on phenotypes, yet accounting for rare variants in genetic analyses is statistically challenging due to the limited number of allele carriers and the burden of multiple testing. While rich variant annotations promise to enable well-powered rare variant association tests, methods integrating variant annotations in a data-driven manner are lacking. Here we propose deep rare variant association testing (DeepRVAT), a model based on set neural networks that learns a trait-agnostic gene impairment score from rare variant annotations and phenotypes, enabling both gene discovery and trait prediction. On 34 quantitative and 63 binary traits, using whole-exome-sequencing data from UK Biobank, we find that DeepRVAT yields substantial gains in gene discoveries and improved detection of individuals at high genetic risk. Finally, we demonstrate how DeepRVAT enables calibrated and computationally efficient rare variant tests at biobank scale, aiding the discovery of genetic risk factors for human disease traits.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="bohm_scalar_2024" class="col-sm-8"> <div class="title">Scalar curvature along Ebin geodesics</div> <div class="author"> Christoph B√∂hm, Timothy Buttsworth, and Brian Clarke </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1515/crelle-2024-0033" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract Let ùëÄ be a smooth, compact manifold and let N Œº }mathcal{N}_{}mu} denote the set of Riemannian metrics on ùëÄ with smooth volume density ùúá. For g 0 ‚àà N Œº g_{0}}in{}mathcal{N}_{}mu}} , we show that if dim ( M ) ‚â• 5 }dim(M)}geq 5 , then there exists an open and dense subset Y g 0 ‚äÇ T g 0 ‚Å¢ N Œº {}mathcal{Y}_{g_{0}}}}subset T_{g_{0}}{}mathcal{N}_{}mu}} (in the C ‚àû C^{}infty} topology) so that, for each h ‚àà Y g 0 h}in{}mathcal{Y}_{g_{0}}} , the ( N Œº , L 2 ) ({}mathcal{N}_{}mu}},L^{2}) Ebin geodesic Œ≥ h ‚Å¢ ( t ) }gamma_{h}(t) with Œ≥ h ‚Å¢ ( 0 ) = g 0 }gamma_{h}(0)=g_{0} and Œ≥ h ‚Ä≤ ‚Å¢ ( 0 ) = h }gamma_{h}^{}prime}(0)=h satisfies lim t ‚Üí ‚àû R ‚Å¢ ( Œ≥ h ‚Å¢ ( t ) ) = ‚àí ‚àû }lim_{t}to}infty}R(}gamma_{h}(t))=-}infty , uniformly, where R : N Œº ‚Üí C ‚àû ‚Å¢ ( M ) R}colon}mathcal{N}_{}mu}}to C^{}infty}(M) is the scalar curvature.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="aivazidis_cell2fate_2025" class="col-sm-8"> <div class="title">Cell2fate infers RNA velocity modules to improve cell fate prediction</div> <div class="author"> Alexander Aivazidis, Fani Memi, Vitalii Kleshchevnikov, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Sezgin Er, Brian Clarke, Oliver Stegle, Omer Ali Bayraktar' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em></em> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41592-025-02608-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract RNA velocity exploits the temporal information contained in spliced and unspliced RNA counts to infer transcriptional dynamics. Existing velocity models often rely on coarse biophysical simplifications or numerical approximations to solve the underlying ordinary differential equations (ODEs), which can compromise accuracy in challenging settings, such as complex or weak transcription rate changes across cellular trajectories. Here we present cell2fate, a formulation of RNA velocity based on a linearization of the velocity ODE, which allows solving a biophysically more accurate model in a fully Bayesian fashion. As a result, cell2fate decomposes the RNA velocity solutions into modules, providing a biophysical connection between RNA velocity and statistical dimensionality reduction. We comprehensively benchmark cell2fate in real-world settings, demonstrating enhanced interpretability and power to reconstruct complex dynamics and weak dynamical signals in rare and mature cell types. Finally, we apply cell2fate to the developing human brain, where we spatially map RNA velocity modules onto the tissue architecture, connecting the spatial organization of tissues with temporal dynamics of transcription.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="ye_predicting_2025" class="col-sm-8"> <div class="title">Predicting functional constraints across evolutionary timescales with phylogeny-informed genomic language models</div> <div class="author"> Chengzhong Ye, Gonzalo Benegas, Carlos Albors, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Jianan Canal Li, Sebastian Prillo, Peter D. Fields, Brian Clarke, Yun S. Song' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.09.21.677619" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract Genomic language models (gLMs) have emerged as a powerful approach for learning genome-wide functional constraints directly from DNA sequences. However, standard gLMs adapted from natural language processing often require extremely large model sizes and computational resources, yet still fall short of classical evolutionary models in predictive tasks. Here, we introduce GPN-Star (Genomic Pretrained Network with Species Tree and Alignment Representation), a biologically grounded gLM featuring a phylogeny-aware architecture that leverages whole-genome alignments and species trees to model evolutionary relationships explicitly. Trained on alignments spanning vertebrate, mammalian, and primate evolutionary timescales, GPN-Star achieves state-of-the-art performance across a wide range of variant effect prediction tasks in both coding and non-coding regions of the human genome. Analyses across timescales reveal task-dependent advantages of modeling more recent versus deeper evolution. To demonstrate its potential to advance human genetics, we show that GPN-Star substantially outperforms prior methods in prioritizing pathogenic and fine-mapped GWAS variants; yields unprecedented enrichments of complex trait heritability; and improves power in rare variant association testing. Extending beyond humans, we train GPN-Star for five model organisms ‚Äì Mus musculus, Gallus gallus, Drosophila melanogaster, Caenorhabditis elegans , and Arabidopsis thaliana ‚Äì demonstrating the robustness and generalizability of the framework. Taken together, these results position GPN-Star as a scalable, powerful, and flexible new tool for genome interpretation, well suited to leverage the growing abundance of comparative genomics data.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> ¬© Copyright 2025 . Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?c15de51d4bb57887caa2c21988d97279"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>